internal/host/static/repository_host.go

```diff
@@ -171,6 +171,16 @@ func (r *Repository) UpdateHost(ctx context.Context, scopeId string, h *Host, ve
 			if err != nil {
 				return errors.Wrap(ctx, err, op)
 			}
+			ha := &hostAgg{
+				PublicId: h.PublicId,
+			}
+			if err := r.reader.LookupByPublicId(ctx, ha); err != nil {
+				if errors.IsNotFoundError(err) {
+					return errors.Wrap(ctx, err, op)
+				}
+				return errors.Wrap(ctx, err, op)
+			}
+			returnedHost.SetIds = ha.GetSetIds()
```

Comment:
We can probably drop this `if errors.IsNotFoundError(err)` and just do a single return on error.  Since we are in a transaction if the update succeeded we should be be able to get a not found, also it doesn't do anything differently from the return outside the if statement.
This is also a good case for adding a custom message to the wrapped error.
```suggestion
			if err := r.reader.LookupByPublicId(ctx, ha); err != nil {
				return errors.Wrap(ctx, err, op, errors.WithMsg("failed to lookup host after update")
			}
			returnedHost.SetIds = ha.GetSetIds()
```